rcl/test/rcl/test_memory.cpp

```diff
@@ -0,0 +1,330 @@
+// Copyright 2015 Open Source Robotics Foundation, Inc.
```

Comment:
```suggestion
// Copyright 2019 Open Source Robotics Foundation, Inc.
```

Or I guess `2018` if you technically wrote it last year, but 2019 I think is fine.